src/demolition_spider.py

```diff
@@ -134,4 +135,5 @@ def extract_records(
                     short_notes=row.xpath("string(.//td[7])").get().strip(),
                 )
             )
+        mail_sender.send_email(records)
```

Comment:
If we can figure out a way to have the spider return the records it extracts, I'd prefer not to have "send an email" functionality in the same file as "crawl the page and extract records" functionality. Is there a straightforward way to make this call from `main.py`?

Reply:
Totally agree, that's the next thing we should try and figure out, I would prefer to use the mail_sender in main, but I'm not seeing a straightforward way to do that at the moment.

Reply:
If nothing else, consider hoisting this call back into `parseResults`